source/install/cloud-changelog.md

```diff
@@ -3,12 +3,41 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-07-28](#release-2021-07-28)
 - [Release 2021-07-15](#release-2021-07-15)
 - [Release 2021-07-01](#release-2021-07-01)
 - [Release 2021-06-16](#release-2021-06-16)
 - [Release 2021-06-02](#release-2021-06-02)
 - [Release 2021-05-21](#release-2021-05-21)
-- [Release 2021-05-05](#release-2021-05-05)
+
+## Release 2021-07-28
+
+### Improvements
+
+#### User Interface (UI)
+ - Improved typing performance in affected environments by reducing the frequency at which drafts are saved.
+ - Improved user and channel selector for app commands.
+ - Added a **Set Status** button if the custom status hasn't changed from currently set status.
+ - Added plugin API methods for user access tokens and OAuth apps.
+ - Improved default rendering of images inserted via the GIF picker.
+ - Small text changes were added to Direct and Group Message menus: 'Mute channel' and 'Edit Channel Header' now reads 'Mute Conversation' and 'Edit Conversation Header'.
+ - Added support for `react-intl` and `<Timestamp/>` usage in plugins.
+
+#### Administration
+ - Fixed some of the incorrect mention counts and unreads around threads and channels since the introduction of collapsed reply threads (Beta). This fix is done through a SQL migration and it may take several minutes to complete for large databases.
```

Comment:
```suggestion
 - Fixed some of the incorrect mention counts and unreads around threads and channels since the introduction of Collapsed Reply Threads (Beta). This fix is done through a SQL migration, and it may take several minutes to complete for large databases.
```